gemfiles/rails_61_webpacker/Gemfile.lock

```diff
@@ -256,14 +256,14 @@ GEM
       net-protocol
     nio4r (2.5.8)
     nio4r (2.5.8-java)
-    nokogiri (1.14.2)
+    nokogiri (1.13.10)
       mini_portile2 (~> 2.8.0)
       racc (~> 1.4)
-    nokogiri (1.14.2-arm64-darwin)
+    nokogiri (1.13.10-arm64-darwin)
       racc (~> 1.4)
-    nokogiri (1.14.2-java)
+    nokogiri (1.13.10-java)
       racc (~> 1.4)
-    nokogiri (1.14.2-x86_64-linux)
+    nokogiri (1.13.10-x86_64-linux)
```

Comment:
Dependabot is still downgrading some gems like Nokogiri when it shouldn't be. I'm not sure why it happens but shows we can't have these auto merge until resolved.